plugins/retina.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: retina
+spec:
+  version: v0.0.2
+  homepage: retina.sh
+  shortDescription: Retina CLI allows the user to capture network traffic and metadata for the capture target, and then send the capture file to the location by Output Configuration.
+  description: |
+    Retina is a cloud-agnostic, open-source Kubernetes network observability platform that provides a centralized hub for monitoring application health, network health, and security. 
+    It provides actionable insights to cluster network administrators, cluster security administrators, and DevOps engineers navigating DevOps, SecOps, and compliance use cases.
+    Retina collects customizable telemetry, which can be exported to multiple storage options (such as Prometheus, Azure Monitor, and other vendors) and visualized in a variety of ways 
+    (like Grafana, Azure Log Analytics, and other vendors). This CLI is a part of the Retina project, which allows the user to capture network traffic and metadata for the capture target.
```

Comment:
wrap to 80 chars, and let's rewrite to be more clear about _what the plugin does_ instead of trying to sell it.
ref [AKS](https://github.com/kubernetes-sigs/krew-index/blob/master/plugins/aks.yaml#L10-L21), [kubectx](https://github.com/kubernetes-sigs/krew-index/blob/master/plugins/ctx.yaml#L10-L11)

Reply:
+1 on this. Keep it lean, less marketing. Focus on concrete functionality.